var/spack/repos/builtin/packages/bfs/package.py

```diff
@@ -6,15 +6,23 @@
 
 
 class Bfs(MakefilePackage):
-    """A breadth-first version of the UNIX find command."""
+    """BFS is a breadth-first variant of the UNIX find command that offers
+    consistent, intuitive behavior and improved performance."""
 
     homepage = "https://github.com/tavianator/bfs"
     url = "https://github.com/tavianator/bfs/archive/refs/tags/3.0.1.tar.gz"
+    git = "https://github.com/tavianator/bfs.git"
+    list_url = "https://github.com/tavianator/bfs/releases"
```

Comment:
Can't we do this in cmd/checksum.py by regex for all GitHub urls? Seems like having to add list_url to all projects which distribute through GitHub archives warrants that.

Reply:
Incidentally this works out of the box without these changes on my system.

Reply:
Interesting I swear I hit something that caused me to add these, but your right today it seems to be locating other releases without them. I'll pull them out for now.